frontend/src/entity/ChangeLog.tsx

```diff
@@ -0,0 +1,6 @@
+import { Change } from './Change';
+
+export interface ChangeLog {
+  changes: Change[];
+  lastViewedAt: string;
```

Comment:
I think this is also optional: https://github.com/short-d/short/blob/master/backend/app/adapter/gqlapi/schema.go#L25